tests/drivers/counter/counter_basic_api/boards/nrf52810_pca10040.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_COUNTER_TIMER0=y
+CONFIG_COUNTER_TIMER0_PRESCALER=4
+CONFIG_COUNTER_TIMER1=y
+CONFIG_COUNTER_TIMER1_PRESCALER=4
+CONFIG_COUNTER_TIMER2=y
+CONFIG_COUNTER_TIMER2_PRESCALER=4
+CONFIG_COUNTER_RTC0=y
```

Comment:
The 52810 only has 2 RTCs

Reply:
Is it intentional to not configure here RTC1 ?

Reply:
RTC1 used by kernel